packages/site/src/pages/index.tsx

```diff
@@ -173,30 +204,52 @@ const Index = () => {
             disabled={!isMetaMaskReady}
           />
         )}
-        {shouldDisplayReconnectButton(state.installedSnap) && (
-          <Card
-            content={{
-              title: 'Reconnect',
-              description:
-                'While connected to a local running snap this button will always be displayed in order to update the snap if a change is made.',
-              button: (
-                <ReconnectButton
-                  onClick={handleConnectClick}
-                  disabled={!state.installedSnap}
-                />
-              ),
-            }}
-            disabled={!state.installedSnap}
-          />
-        )}
         <Card
           content={{
-            title: 'Send Hello message',
+            title: 'Get public key',
+            description:
+              'Get the public key of the installed snap. Log on the console.',
```

Comment:
Question: This description sounds like there's only one public key and one snap allowed - are both of those assumptions correct?

Reply:
Yes. This testing site uses a fixed BIP-32 derivation path; hence, we have a single public key.

Reply:
Marking as resolved.